src/plugins/unified_search/public/typeahead/index.tsx

```diff
@@ -7,8 +7,13 @@
  */
 
 import React from 'react';
+import { EuiDelayRender, EuiLoadingSpinner } from '@elastic/eui';
 
-const Fallback = () => <div />;
+const Fallback = () => (
```

Comment:
is it related to that PR?

Reply:
This thing doesn't show anything. I decided to change it to look like in another part of the application. to show the spinner

Reply:
can we use i`mport { withSuspense } from '@kbn/shared-ux-utility';` instead?